src/rgw/setup/rgw.py

```diff
@@ -144,15 +144,26 @@ def start(conf: MappedConf, index: str):
 
         Log.info(f'Configure logrotate for {const.COMPONENT_NAME} at path: {const.LOGROTATE_CONF}')
         Rgw._logrotate_generic(conf)
+
         # Before starting service,Verify backend store value=motr in rgw config file.
         Rgw._verify_backend_store_value(conf)
+
+        # Create motr trace & addb stob dirctory.
+        # Collect fid value of motr to create addb value.
+        config_file = Rgw._get_rgw_config_path(conf)
+        confstore_url = const.CONFSTORE_FILE_HANDLER + config_file
+        Rgw._load_rgw_config(Rgw._conf_idx, confstore_url)
+        motr_fid_key = const.MOTR_MY_FID % index
+        motr_fid_value = Conf.get(Rgw._conf_idx, motr_fid_key)
         log_path = Rgw._get_log_dir_path(conf)
         motr_trace_dir = os.path.join(log_path, 'motr_trace_files')
+        addb_dir = os.path.join(log_path, f'addb_files-{motr_fid_value}')
```

Comment:
how rgw picks up this location to store addb files to this location, there should be some corresponding config right?

Reply:
we are creating a addb stob path in this PR.
as a part of [CORTX-29728](https://jts.seagate.com/browse/CORTX-29728), we will enable addb log in rgw using config parameter (motr addb enabled = true).

Reply:
sure as discussed, confirm with Motr team to understand if it can cause issue if we enable this config right away. If error handling is there in libMotr, it should not matter with RGW startup ideally

Reply:
lets target this separately only as planned, as there is no reply from Motr team